docs/examples/overlaypanel/confirmationExample.js

```diff
@@ -0,0 +1,193 @@
+// @flow strict
+import React, { type Node, useCallback } from 'react';
+import {
+  Box,
+  Button,
+  CompositeZIndex,
+  FixedZIndex,
+  Flex,
+  Layer,
+  OverlayPanel,
+  Text,
+  TextField,
+  Fieldset,
+  Checkbox,
+  RadioButton,
+} from 'gestalt';
+
+export default function Example(): Node {
+  const [shouldShow, setShouldShow] = React.useState(false);
+  const HEADER_ZINDEX = new FixedZIndex(10);
+  const sheetZIndex = new CompositeZIndex([HEADER_ZINDEX]);
+
+  const renderFooter = useCallback(
+    ({ onDismissStart }) => (
+      <Flex alignItems="center" justifyContent="end">
+        <Button color="red" text="Create" onClick={onDismissStart} />
+      </Flex>
+    ),
+    [],
+  );
+
+  return (
+    <React.Fragment>
+      <Box padding={8}>
+        <Button text="View example OverlayPanel" onClick={() => setShouldShow(true)} />
+      </Box>
+      {shouldShow && (
+        <Layer zIndex={sheetZIndex}>
+          <OverlayPanel
+            dismissConfirmation={{}}
+            accessibilityDismissButtonLabel="Close audience creation overlaypanel"
+            accessibilityOverlayPanelLabel="Audience list creation for new campaign"
+            heading="Create a new audience list"
+            onDismiss={() => setShouldShow(false)}
+            footer={renderFooter}
+            size="md"
+          >
+            <Flex
+              direction="column"
+              gap={{
```

Comment:
Do these Flex's need different values for row and column? ("yes" is a totally valid answer, just double-checking)